### PR TITLE
Implement Solidjs

### LINKED ISSUE #3: Implement solidjs
Reference: https://www.solidjs.com/

- Please use `requestAnimationFrame` instead of `setTimeout` or `setInterval`

### PR DESCRIPTION
**Pull request by pontakornth**
Resolve #3

Most code are translation from React Hook implementation. I include some comment in case someone wants to compare.

### CODE REVIEW

**Review by elianiva — APPROVED** [review]

File: solidjs/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <link rel="shortcut icon" type="image/ico" href="/assets/favicon.0e726a38.ico" />
+    <title>Solid: Decimal Clock</title>
+    <script type="module" crossorigin src="/assets/index.ad505d05.js"></script>
```
Comment (ronnygunawan) [inline solidjs/dist/index.html:9]: Please make sure generated index.html uses relative path

**Review by ronnygunawan — CHANGES_REQUESTED** [review]
Please configure the build to make sure that the generated index.html uses relative path when linking to .js and .css files. Everything else LGTM

**Comment by pontakornth** [comment]
I think this should be alright.

**Review by ronnygunawan — APPROVED** [review]
LGTM

**Pull request merged**